src/main/topLevelPages/myDashboard/newVirtualContributorWizard/LoadingState.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import DialogHeader from '../../../../core/ui/dialog/DialogHeader';
+import Loading from '../../../../core/ui/loading/Loading';
+import { Caption } from '../../../../core/ui/typography';
+import Gutters from '../../../../core/ui/grid/Gutters';
+import { gutters } from '../../../../core/ui/grid/utils';
+import CancelDialog from './CancelDialog';
+
+type LoadingStateProps = {
+  onClose: () => void;
+};
+
+const LoadingState = ({ onClose }: LoadingStateProps) => {
+  const { t } = useTranslation();
+  const [dialogOpen, setDialogOpen] = useState(false);
+
+  const onCancel = () => {
+    setDialogOpen(true);
+  };
+
+  return (
+    <>
+      <DialogHeader onClose={onCancel} />
+      <Loading text="" />
+      <Gutters padding={gutters(2)} sx={{ textAlign: 'center' }}>
```

Comment:
I think `textAlign` is a property of `BoxProps` and can be used directly outside `sx`